arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -205,6 +205,35 @@ bool RestReplicationHandler::isCoordinatorError() {
   return false;
 }
 
+std::string RestReplicationHandler::LoggerState = "logger-state";
```

Comment:
these should also be defined as const std::string s.

Reply:
yap